README.md

```diff
@@ -108,6 +120,15 @@ In order to connect your ragtime bot to your Discord server, follow these steps:
 - Select ‘Read Message’, ‘Send Message’, and ‘Manage Message’ options.
 - Copy the generated URL and paste it into your web browser. It will redirect you to the Discord page to add the bot to your server.
 
+
+## How to setup whatsapp bot using twilio
```

Comment:
Are there any links to twilio docs that can show how to setup a paid sandbox (like we have), so that users don't have to sign-in to the sandbox every time?